<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="3">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTreeView" name="treeView"/>
     <widget class="QTableView" name="tableView">
      <property name="font">
       <font>
        <family>Courier New</family>
       </font>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="openFilePushButton">
     <property name="text">
      <string>Open file ...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="updateShapePushButton">
     <property name="text">
      <string>Update shape</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="onUpdateStageButton">
     <property name="text">
      <string>Update stage</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>openFilePushButton</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>onOpenFileButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>updateShapePushButton</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>onAddShapeButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>584</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>66</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>onUpdateStageButton</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>onUpdateStageButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onOpenFileButton()</slot>
  <slot>onAddShapeButton()</slot>
  <slot>onUpdateStageButton()</slot>
 </slots>
</ui>
